app/components/modal-nfc/modal-nfc-flashcard.tsx

```diff
@@ -31,6 +31,7 @@ export const ModalNfcFlashcard: React.FC<{
     setIsActive(false)
     if (nfcRegistered) {
       try {
+        await NfcManager.cancelTechnologyRequest()
```

Comment:
Why are you calling cancelTechnologyRequest() method twice here? I think you can remove one.

Reply:
done